lib/mic.js

```diff
@@ -13,6 +13,7 @@ var mic = function mic(options) {
     var channels = options.channels || '1';
     var device = options.device || 'plughw:1,0';
     var exitOnSilence = options.exitOnSilence || 0;
+    var fileType = options.fileType || 'raw';
```

Comment:
Thanks for the pull request, is it possible to even update the README.md file so that the new options are clearly documented in the API for a new user?